cobe/pmodule/pmodule.py

```diff
@@ -0,0 +1,66 @@
+import os
+from time import sleep
+import psutil
+import subprocess
+
+root_folder = "C:\\Users\\David\\Documents\\predprey_batches"
+docker_path = "C:\\Program Files\\Docker\\Docker\\Docker Desktop.exe"
+
+def is_file_running(name: str) -> bool:
```

Comment:
duplicate of renderingstack class method, should be refactored into shared function in tools